widgetastic_manageiq.py

```diff
@@ -1589,8 +1589,9 @@ def __repr__(self):
             params = ", ".join(["{}={}".format(attr, getattr(self, attr)) for attr in attrs])
             return "TimelinesEvent({})".format(params)
 
-    def __init__(self, parent, logger=None):
+    def __init__(self, parent, locator=None, logger=None):
         super(TimelinesChart, self).__init__(parent=parent, logger=logger)
+        self.locator = locator or '//div[contains(@class, "timeline-container")]'
```

Comment:
@mshriver @otsuman to save time, I would personally keep it here for now and then backport it to PF, then remove and bump ...